_test/test_mpi/test_ParpoolMPI_Framework.m

```diff
@@ -53,41 +53,36 @@
             else
                 obj.pool = pl;
             end
-            
+
         end
         function delete(obj)
             obj.pool_deleter = [];
         end
         %
         function test_finish_tasks_reduce_messages(obj,varargin)
             if obj.ignore_test
-                return;
+                skipTest();
```

Comment:
Might we put "_Parallel computing toolbox is not available_" as the reason for skipping this, and the following, tests?